gitpod-io/website
# Fix anchor target scroll position

## Description
This fixes anchor target elements which are concealed by the navbar. I haven't fully tested on all modern browsers (except Chrome and Firefox). So, if someone can help test it out especially on Edge and/or Safari that would be much appreciated. Let me know your thoughts.

## Related Issue(s)
Fixes #987

## How to test

1. Go to Docs Page
2. Click "Your computer in the cloud"
3. Notice that "Your computer in the cloud" text is not concealed by the navbar


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1016"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
